Remora.Discord.Extensions/Formatting/Markdown.cs

```diff
@@ -0,0 +1,200 @@
+
+        /// <summary>
+        /// Formats a string to use Markdown Spoiler formatting.
+        /// </summary>
+        /// <param name="text">The input text to format.</param>
+        /// <returns>
+        /// A markdown-formatted spoilered string.
+        /// </returns>
+        public static string Spoiler(string text) => $"||{text}||";
+
+        /// <summary>
+        /// Formats a string to use Markdown Inline Code formatting.
+        /// </summary>
+        /// <param name="text">The input text to format.</param>
+        /// <returns>
+        /// A markdown-formatted inline code string.
+        /// </returns>
+        public static string InlineCode(string text) => $"`{text}`";
+
+        /// <summary>
+        /// Formats a string to use Markdown Block Code formatting.
+        /// </summary>
+        /// <param name="text">The input text to format.</param>
+        /// <returns>
+        /// A markdown-formatted block code string.
+        /// </returns>
+        public static string BlockCode(string text) => BlockCode(text, string.Empty);
+
+        /// <summary>
+        /// Formats a string to use Markdown Block Code formatting with a specified
+        /// language for syntax highlighting.
+        /// </summary>
+        /// <remarks>
+        /// Discord supports the following languages: <see href="https://github.com/highlightjs/highlight.js/tree/main/src/languages"/>.
+        /// </remarks>
+        /// <param name="text">The input text to format.</param>
+        /// <param name="language">The language.</param>
+        /// <returns>
+        /// A markdown-formatted block code string.
+        /// </returns>
+        public static string BlockCode(string text, string language) => $"```{language}\n{text}\n```";
+
+        /// <summary>
+        /// Formats a string to use Markdown Quote Block formatting.
+        /// </summary>
+        /// <param name="text">The input text to format.</param>
+        /// <returns>
+        /// A markdown-formatted block quote string.
+        /// </returns>
+        public static string BlockQuote(string text) => $">>> {text}\n";
+
+        /// <summary>
+        /// Escapes a URL to prevent embedding of its content.
+        /// </summary>
+        /// <param name="uri">The URI to escape.</param>
+        /// <returns>
+        /// A formatted URL string that will not embed its content.
+        /// </returns>
+        public static string EscapeUrl(Uri uri) => EscapeUrl(uri.ToString());
+
+        /// <summary>
+        /// Escapes a URL to prevent embedding of its content.
+        /// </summary>
+        /// <param name="url">The URL to escape.</param>
+        /// <returns>
+        /// A formatted URL string that will not embed its content.
+        /// </returns>
+        public static string EscapeUrl(string url) => $"<{url}>";
+
+        /// <summary>
+        /// Formats a string to contain a Markdown hyperlink of itself. The hyperlink will only hyperlink inside
```

Comment:
Webhooks (e.g interactions) also have support for hyperlinks.

Reply:
What do you mean? How would that affect this use case?